pkg/secrethub/secret.go

```diff
@@ -171,7 +171,7 @@ func (s secretService) Write(path string, data []byte) (*api.SecretVersion, erro
 	}
 
 	key, err := s.client.getSecretKey(secretPath)
-	if err == api.ErrSecretNotFound {
+	if api.IsErrNotFound(err) {
```

Comment:
`api.ErrNoOKSecretKey` appears to be a not found error as well. I believe we should check for a specific error in this case.